task.sh

```diff
@@ -6,38 +6,40 @@
 #   dir2
 #   dir3
 #       dir4
-
+mkdir ./task ./task/dir1 ./task/dir2 ./task/dir3 ./task/dir3/dir4
```

Comment:
`./` можно не использовать везде, кроме 34 строчки
```suggestion
mkdir task task/dir1 task/dir2 task/dir3 task/dir3/dir4
```